openshift/post-deploy-tests.template.yaml

```diff
@@ -25,7 +25,12 @@ objects:
             imagePullPolicy: IfNotPresent
             name: gabi-post-deploy-tests
             command:
-              - /bin/date
+              - /bin/curl
+            args:
+              - --silent
+              - --show-error
+              - --fail
+              - http://$(HOST):8080/healthcheck
```

Comment:
**To the reviewer:**

This aims to act as a smoke test for all intents and purposes before a more sophisticated acceptance test can be added.